public/css/style.css

```diff
@@ -29,21 +30,84 @@ body {
 }
 
 .headerImgContainer {
-	height: 100%;
+	height: 100px;
 }
 
-.headerMenuArrow {
-	position: absolute;
-	height: 0px;
-	width: 0px;
+/*****filter container *****/
+
+.filterName {
+	text-align: center;
+	font-size: 2em;
+	color: #b4b4b4;
+	width:100%;
+}
+
+.clearButton {
+	width:35px;
+	height:35px;
+	padding-left: 10px;
 }
 
 /**********************account menu ****************************/
 .accountMenuContainer {
-	position: absolute;
-	width:0px;
-	height:0px;
-	overflow:hidden;
+	position: fixed;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	top: 175px;
+	left: 20px;
+	right: 20px;
+	z-index: 3;
+	color: #c5c5c5;
+	background-color: #42474B;
+	border-radius: 15px;
+	transition: right 0.5s ease-in-out, top 0.5s ease-in-out;
+}
+
+.title {
+    width: 95%;
```

Comment:
Be consistent with your spacing, not just within a single block, but for all of your code in all files.  One style to rule them all.